src/main/java/bio/terra/service/profile/ProfileDao.java

```diff
@@ -98,8 +98,12 @@ public BillingProfileModel createBillingProfile(BillingProfileRequestModel profi
             .billingAccountId(keyHolder.getString("billing_account_id"))
             .description(keyHolder.getString("description"))
             .cloudPlatform(CloudPlatform.valueOf(keyHolder.getString("cloud_platform")))
-            .tenantId(keyHolder.getString("tenant_id"))
-            .subscriptionId(keyHolder.getString("subscription_id"))
+            .tenantId(Optional.ofNullable(keyHolder.getField("tenant_id", UUID.class))
```

Comment:
This is ok, but it would be better to have `getField()` return `Optional<T>` instead of `T`. I'm now thinking every time we write `Optional.ofNullable()` and it's not in a `return` statement, we should try to update the code to move it to a `return`, if it's in our source code.